devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -98,15 +155,17 @@ def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelCmdEncap
 		cmd.command = cmd.parameter[1]
 		cmd.parameter = cmd.parameter.drop(2)
 	}
-	def encapsulatedCommand = cmd.encapsulatedCommand()
+    
+	def encapsulatedCommand = cmd.encapsulatedCommand([0x60: 3, 0x79: 1])
```

Comment:
Can you remove the `0x79: 1` you're passing in?

We haven't updated our backend to support parsing of Sound Switch yet (and there's no handler established in this DTH anyway) so it's not actually doing anything.

Reply:
@greens - Got it, I've removed it from the encapsulatedCommand and secureEncapsulatedCommand.